### PR TITLE
csv2tsv: add page

### PR DESCRIPTION
**Pull request by corneliusroemer**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.2.1

### CODE REVIEW

File: pages/common/csv2tsv.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# csv2tsv
+
+> Convert CSV (comma-separated) text to TSV (tab-separated) format.
+> More information: <https://github.com/eBay/tsv-utils/blob/master/README.md#csv2tsv>.
+
+- Convert from CSV to TSV:
+
+`csv2tsv {{path/to/input_csv1 path/to/input_csv2 ...}} > {{path/to/output_tsv}}`
+
```
Comment (kbdharun) [inline pages/common/csv2tsv.md:9]: I have a suggestion, this [page](https://github.com/eBay/tsv-utils/blob/master/docs/tool_reference/csv2tsv.md) specifies there are multiple field delimiters present like comma(default), etc. So maybe we can add a generalized command while having the `semicolon` example.
```suggestion

- Convert field delimiter separated CSV to TSV:

`csv2tsv -c'{{field_delimiter}}' {{path/to/input_csv}}`

```

**Comment by corneliusroemer** [comment]
Great point @kbdharun I committed your suggestion

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks @corneliusroemer , Let's see what others think.

**Review by einverne — APPROVED** [review]
Looks good to me.

**Pull request merged**